modules/hal_nordic/nrfs/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
```

Comment:
Please move this under `if(CONFIG_NRFS)` to prevent the following CMake warning:
```
  No SOURCES given to Zephyr library: modules__hal_nordic__nrfs
```
when NRFS is not enabled.